.github/workflows/release.yml

```diff
@@ -93,3 +99,22 @@ jobs:
           echo "scm.url=scm\:git\:git@github.com\:${GITHUB_REPOSITORY}.git" > release.properties
           echo "scm.tag=${{ github.event.inputs.perform_version }}" >> release.properties
           mvn ${MAVEN_FLAGS} release:perform
+      - name: Send Repository Dispatch Event
+        # We not send any repository dispatch if parent_version and perform_version set
+        if: success() && github.event.inputs.send_repository_dispatch == 'yes' && github.event.inputs.parent_version == '' && github.event.inputs.perform_version == ''
```

Comment:
I don't think the `github.event.inputs.parent_version == ''` check is necessary. Even if the oss-parent version was updated here, we want to trigger the release of oss-parent (with the new version of commons).

I am also wondering if we shouldn't just check for `success() && github.event.inputs.send_repository_dispatch == 'yes'`. If `release:prepare` is successful (new tag pushed) but `release:perform` fails (e.g.m sonatype timeout), this is when we re-run and specify `perform_version`. In that case, we will want the oss-parent to be released with the new version of commons.

Reply:
Yeah you're right. We have `github.event.inputs.send_repository_dispatch == 'yes'`, and let this alone to be a control whether we want to send repository_dispatch or not.